src/Microsoft.Security.Utilities.Core/UnclassifiedPotentialSecurityKeys/SEC101_061_LooseOAuth2BearerToken.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Security.Utilities
+{
+    public class OAuth2BearerToken : RegexPattern
```

Comment:
This needs to be added to `WellKnownRegexPatterns.UnclassifiedPotentialSecurityKeys`. Pushing this point a bit more, perhaps it would be nice to autogenerate the list?